subsys/bluetooth/nrf_bt_adv/providers/Kconfig.swift_pair

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NRF_BT_ADV_PROV_SWIFT_PAIR
+	bool "Microsoft Swift Pair"
+	help
+	  Adds Microsoft Swfit Pair manufacturer data to advertising data if
+	  used Bluetooth local identity has no bonds and advertising has not
+	  entered grace period.
+
+if NRF_BT_ADV_PROV_SWIFT_PAIR
+
+config NRF_BT_ADV_PROV_SWIFT_PAIR_COOL_DOWN_DURATION
+	int "Swift Pair cool down duration [s]"
+	default 30
+	range 30 600
+	help
+	  To ensure users would not try to connect to a device which is no
+	  longer available, Swift Pair vendor section is removed and
+	  advertising is continued without Swift Pair data for additional time
+	  before the advertising is stopped.
+
+	  The cool down phase is reported as requested grace period.
```

Comment:
grace period, which is not shorter then requested value. Grace period can be longer.